examples/storage/src/main.rs

```diff
@@ -0,0 +1,99 @@
+#![no_main]
+#![no_std]
+#![feature(impl_trait_in_assoc_type)]
+#![feature(used_with_arg)]
+
+use riot_rs::debug::log::{defmt, info};
+
+use riot_rs::storage;
+
+use serde::{Deserialize, Serialize};
```

Comment:
I grouped the storage specific imports and added a comment. Is that fine (even though violating the CC)?

I prefer the showcase imports to stand out.

Reply:
... because time and again I'm wondering why this isn't done usually. The point here is to explain readers how to use the storage module. Seeing these imports bundled with a comment clarifies things.